modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/datatypes/NumericUpdateSourcesCoercionTest.java

```diff
@@ -287,7 +287,7 @@ private static Stream<Arguments> argsForUpdateWithLiteralValue() {
                         .opMatches(castTo(NativeTypes.INT32)),
 
                 forTypePair(NumericPair.BIGINT_BIGINT)
-                        .opMatches(castTo(NativeTypes.INT64)),
+                        .opMatches(ofTypeWithoutCast(NativeTypes.INT64)),
```

Comment:
also need to fix 
``argsForUpdateWithColumnAsValue`` and ``argsDyn`` methods

Reply:
done